nix/overlays/required.nix

```diff
@@ -0,0 +1,34 @@
+{ pkgs, enableProfiling }:
+
+with import ../../lib.nix;
+
+with pkgs.haskell.lib;
+
+let
+  addRealTimeTestLogs = drv: overrideCabal drv (attrs: {
+    testTarget = "--show-details=streaming";
+  });
+in
+
+self: super: {
+
+    ########################################################################
+    # Overides of Cardano SL packages
+
+    ########################################################################
+    language-plutus-core = addRealTimeTestLogs super.language-plutus-core;
+    # The base Haskell package builder
+
+    mkDerivation = args: super.mkDerivation (args // {
+      enableLibraryProfiling = enableProfiling;
+      enableExecutableProfiling = enableProfiling;
+      splitCheck = true;
+      # Static linking for everything to work around
+      # https://ghc.haskell.org/trac/ghc/ticket/14444
+      # This will be the default in nixpkgs since
+      # https://github.com/NixOS/nixpkgs/issues/29011
+      enableSharedExecutables = false;
```

Comment:
Since it is now unlikely that we use a version of `nixpkgs` that has not `enableSharedExecutables = false`  by default, we can remove this line and above comment?

Reply:
Yeah, this seems true for 18.09